agent_factory/integrations/telegram/github_handlers.py

```
@@ -0,0 +1,503 @@
+            capture_output=True,
+            text=True
+        )
+
+        await context.bot.send_message(
+            chat_id=chat_id,
+            text="📝 Committed!"
+        )
+
+        # Push
+        subprocess.run(
+            ["git", "push", "origin", "main"],
```

Ruff (0.14.8)

Starting a process with a partial executable path

(S607)